app/scripts/directives/collapsewhen.js

```diff
@@ -1,43 +1,65 @@
 'use strict';
 
-SwaggerEditor.directive('collapseWhen', function () {
-  var TRANSITION_DURATION = 200; //ms
+SwaggerEditor.directive('collapseWhen', ['$interval', function ($interval) {
```

Comment:
Hi, I am using $interval service and to define it, we have to inject into our directive, otherwise it will not work. 
I have found ngAnnotate tasks, so it can be simplified. I've added new commit https://github.com/swagger-api/swagger-editor/commit/71155e3d7294b9a405261bde737322cd6a34a78e